server/application/application.proto

```diff
@@ -94,6 +95,11 @@ service ApplicationService {
 		option (google.api.http).get = "/api/v1/applications/{name}";
 	}
 
+        // GetManifests returns application manifests
+	rpc GetManifests(ApplicationQuery) returns (repository.ManifestResponse) {
```

Comment:
We need a new ManifestQuery type instead of ApplicationQuery to handle different querying fields like revision.

Reply:
@jessesuen have added this type.  Can test with:

    curl -k 'http://localhost:8080/api/v1/applications/guestbook/manifests?revision=7ad56707102a31d64214f8fb47ab840fd2550826' | jq